tests/ftests/066-sudo-systemd_cgclassify-v2.py

```diff
@@ -0,0 +1,213 @@
+    if not Systemd.create_and_validate(config, CONTROLLER, OTHER_CGNAME, ignore_systemd=True):
+        result = consts.TEST_FAILED
+        cause = (
+                    'Failed to create cgroup {} under '
+                    '/sys/fs/cgroup/{}/'.format(OTHER_CGNAME, CONTROLLER)
+                )
+
+    return result, cause
+
+
+def create_process_get_pid(config, CGNAME, SLICE='', ignore_systemd=False):
+    result = consts.TEST_PASSED
+    cause = None
+
+    config.process.create_process_in_cgroup(
+                                                config, CONTROLLER, CGNAME,
+                                                ignore_systemd=ignore_systemd
+                                            )
+
+    pids = Cgroup.get_pids_in_cgroup(config, os.path.join(SLICE, CGNAME), CONTROLLER)
+    if pids is None:
+        result = consts.TEST_FAILED
+        cause = 'No processes were found in cgroup {}'.format(CGNAME)
+
+    return pids, result, cause
+
+
+def terminate_process(config, pids):
+    if pids:
+        for p in pids.splitlines():
+            Run.run(['sudo', 'kill', '-9', p])
+
+
+def test(config):
+    global SYSTEMD_PIDS, OTHER_PIDS
+
+    result = consts.TEST_PASSED
+    cause = None
+
+    # Test cgclassify, that creates a process and then uses cgclassify
+    # to migrate the task the cgroup.
+    SYSTEMD_PIDS, result, cause = create_process_get_pid(
+                                                            config, SYSTEMD_CGNAME,
+                                                            os.path.join(SLICE, SCOPE)
+                                                        )
+    if result == consts.TEST_FAILED:
+        return result, cause
+
+    OTHER_PIDS, result, tmp_cause = create_process_get_pid(
+                                                                config, OTHER_CGNAME,
+                                                                ignore_systemd=True
+                                                          )
+    if result == consts.TEST_FAILED:
+        return result, cause
+
+    # SYSTEMD_CGNAME already has the pid of the task, that scope was created
+    # with and killing it will remove the scope, be careful and pick the newly
+    # spawned task
+    SYSTEMD_PIDS = SYSTEMD_PIDS.split('\n')[1]
+
+    # classify the task from the non-systemd scope cgroup to systemd scope
+    # cgroup (should fail due to wrong cgroup path).
+    try:
+        Cgroup.classify(config, CONTROLLER, SYSTEMD_CGNAME, OTHER_PIDS, ignore_systemd=True)
+    except RunError as re:
+        err_str = 'Error changing group of pid {}: No such file or directory'.format(OTHER_PIDS)
+        if re.stderr != err_str:
+            raise re
+    else:
+        result = consts.TEST_FAILED
+        cause = 'Changing group of pid {} erroneously succeeded'.format(OTHER_PIDS)
+
+    # classify the task from the non-systemd scope cgroup to systemd scope
+    # cgroup (should fail due to wrong cgroup path).
```

Comment:
nitpick - same comment as on the previous test.  I think this comment is a copy/paste error :)